src/redux/modules/lessons.js

```diff
@@ -42,6 +42,7 @@ export const actions = {
 
 const initialState = {
   seriesIDs: [],
+  seriesLoaded: false,
```

Comment:
Are you don't need fetch series on language change? If yes need, you can look here same functionality
[redux](https://github.com/Bnei-Baruch/kmedia-mdb/blob/b8c3160e6b407857bc9dd65318f6d0eac6ce1b99/src/redux/modules/preparePage.js#L32)
[saga](https://github.com/Bnei-Baruch/kmedia-mdb/blob/b8c3160e6b407857bc9dd65318f6d0eac6ce1b99/src/sagas/preparePage.js#L14)